packages/vscode/package.json

```diff
@@ -61,14 +61,30 @@
           "source.js",
           "source.ts",
           "source.js.jsx",
+          "source.jsx",
           "source.tsx",
           "source.vue"
         ],
         "scopeName": "inline.prisma",
-        "path": "./syntaxes/prisma.js.json",
+        "path": "./syntaxes/prisma-inlined.json",
         "embeddedLanguages": {
           "meta.embedded.block.prisma": "prisma"
         }
+      },
+      {
+        "injectTo": [
+          "source.js",
+          "source.ts",
+          "source.js.jsx",
+          "source.jsx",
+          "source.tsx",
+          "source.vue"
+        ],
+        "scopeName": "client.rawsql.prisma",
+        "path": "./syntaxes/prisma-client-rawsql.json",
+        "embeddedLanguages": {
+          "meta.embedded.block.sql": "sql"
+        }
```

Comment:
Is this the magic sauce, and the rest just plumbing around it? 🤯

Reply:
Yeah https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#embedded-languages

> If your grammar includes embedded languages within the parent language, such as CSS style blocks in HTML, you can use the embeddedLanguages contribution point to tell VS Code to treat the embedded language as distinct from the parent language. This ensures that bracket matching, commenting, and other basic language features work as expected in the embedded language.
The embeddedLanguages contribution point maps a scope in the embedded language to a top-level language scope.